integration-tests-scala/src/test/scala/tech/beshu/ror/integration/suites/AdminApiSuite.scala

```diff
@@ -14,47 +14,67 @@
  *    You should have received a copy of the GNU General Public License
  *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
  */
-package tech.beshu.ror.integration.plugin
+package tech.beshu.ror.integration.suites
 
+import cats.data.NonEmptyList
 import com.dimafeng.testcontainers.{ForAllTestContainer, MultipleContainers}
 import org.apache.commons.lang.StringEscapeUtils.escapeJava
 import org.scalatest.Matchers._
 import org.scalatest.{BeforeAndAfterEach, WordSpec}
-import tech.beshu.ror.integration.plugin.AdminApiTests.{insertInIndexConfig, removeConfigIndex}
-import tech.beshu.ror.utils.containers.ReadonlyRestEsCluster.AdditionalClusterSettings
-import tech.beshu.ror.utils.containers.{ElasticsearchNodeDataInitializer, ReadonlyRestEsCluster}
+import tech.beshu.ror.integration.suites.AdminApiSuite._
+import tech.beshu.ror.utils.containers.generic._
 import tech.beshu.ror.utils.elasticsearch.{ActionManagerJ, DocumentManagerJ, IndexManagerJ, SearchManager}
 import tech.beshu.ror.utils.httpclient.RestClient
 import tech.beshu.ror.utils.misc.Resources.getResourceContent
 
-class AdminApiTests extends WordSpec with ForAllTestContainer with BeforeAndAfterEach {
+trait AdminApiSuite
+  extends WordSpec
+    with ForAllTestContainer
+    with EsClusterProvider
+    with BeforeAndAfterEach
+    with MultipleEsTargets
+    with MultipleClients {
+  this: EsContainerCreator =>
 
-  private lazy val rorWithIndexConfig = ReadonlyRestEsCluster.createLocalClusterContainer(
-    name = "ROR1",
-    rorConfigFileName = "/admin_api/readonlyrest.yml",
-    clusterSettings = AdditionalClusterSettings(
+  val rorConfigFileName = "/admin_api/readonlyrest.yml"
+
+  private lazy val rorWithIndexConfig = createLocalClusterContainer(
+    EsClusterSettings(
+      name = "ROR1",
+      rorConfigFileName = rorConfigFileName,
       numberOfInstances = 2,
-      nodeDataInitializer = AdminApiTests.nodeDataInitializer()
+      nodeDataInitializer = AdminApiSuite.nodeDataInitializer()
     )
   )
 
-  private lazy val rorWithNoIndexConfig = ReadonlyRestEsCluster.createLocalClusterContainer(
-    name = "ROR2",
-    rorConfigFileName = "/admin_api/readonlyrest.yml",
-    clusterSettings = AdditionalClusterSettings(configHotReloadingEnabled = false)
+  private lazy val rorWithNoIndexConfig = createLocalClusterContainer(
+    EsClusterSettings(
+      name = "ROR2",
+      rorConfigFileName = rorConfigFileName,
+      configHotReloadingEnabled = false)
   )
 
-  override lazy val container: MultipleContainers = MultipleContainers(rorWithIndexConfig, rorWithNoIndexConfig)
+  private lazy val ror1_1Node = rorWithIndexConfig.nodesContainers.head
+  private lazy val ror1_2Node = rorWithIndexConfig.nodesContainers.tail.head
+  private lazy val ror2_1Node = rorWithNoIndexConfig.nodesContainers.head
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'ror2_1Node' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=5163287)